### PR TITLE
Add FormParser for session key view

### PR DESCRIPTION
**Pull request by kirk444**
Session key view, for retrieving session key, requires urlencoded data to be sent in the request, and this is only parsed properly with the FormParser

# Pull Request

## Related Issue
#47 

## New Behavior
Add FormParser for urlencoded data

---

## Contrast to Current Behavior
No form parser, so no urlencoded data is parsed

---

## Discussion: Benefits and Drawbacks
This change allows retrieving session keys via API.

---

## Changes to the Documentation
Documentation already reflects this (currently broken) functionality

---

## Double Check

* [x ] I have explained my PR according to the information in the comments
 or in a linked issue.
* [x ] My PR targets the `dev` branch.

### CODE REVIEW

**Comment by abhi1693** [comment]
@kirk444 Your changes are failing on the tests. Could you please check the code again?

**Comment by kirk444** [comment]
It seems like maybe the test wants it to also have the multi parse, I will put in a new pull request that adds that as well, I guess

File: netbox_secrets/api/views.py [hunk 1]
```diff
@@ -151,6 +152,8 @@ class GetSessionKeyViewSet(ViewSet):
 
     permission_classes = [IsAuthenticated]
 
+    parser_classes = [ JSONParser, FormParser ]
```
Comment (uedvt359) [inline netbox_secrets/api/views.py:155]:
```suggestion
    parser_classes = [JSONParser, FormParser, MultiPartParser]
```

File: netbox_secrets/api/views.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 from drf_yasg.utils import swagger_auto_schema
 from netbox.api.viewsets import NetBoxModelViewSet
 from rest_framework.exceptions import ValidationError
+from rest_framework.parsers import  JSONParser, FormParser
```
Comment (uedvt359) [inline netbox_secrets/api/views.py:10]:
```suggestion
from rest_framework.parsers import  JSONParser, FormParser, MultiPartParser
```

**Review by uedvt359 — COMMENTED** [review]
these changes should fix the tests

**Pull request merged**